OamLog+services.yaml

```diff
@@ -30,6 +30,8 @@ servers:
           uuid: ol-1-0-0-op-s-1000
     service:
       basic:
+        - service-name: /v1/start-application-in-generic-representation
+          uuid: ol-1-0-0-op-s-2000
         - service-name: /v1/inform-about-application
           uuid: ol-1-0-0-op-s-2001
```

Comment:
As per my understanding, /v1/redirect-oam-request-information (operation-client under each application) is applicable only for OL application, moving this service to "individual management (im)" would be more precise.

Reply:
Changed as proposed.